packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -307,10 +307,8 @@ export async function buildSwingset(
  * @property {ReturnType<typeof import('@agoric/swing-store').makeArchiveSnapshot>} [archiveSnapshot]
  * @property {ReturnType<typeof import('@agoric/swing-store').makeArchiveTranscript>} [archiveTranscript]
  * @property {() => object | Promise<object>} [afterCommitCallback]
- * @property {import('./chain-main.js').CosmosSwingsetConfig} swingsetConfig
- *   TODO refactor to clarify relationship vs. import('@agoric/swingset-vat').SwingSetConfig
- *   --- maybe partition into in-consensus "config" vs. consensus-independent "options"?
- *   (which would mostly just require `bundleCachePath` to become a `buildSwingset` input)
+ * @property {number} [maxVatsOnline]
+ * @property {boolean} [restartWorkerOnSnapshot]
```

Comment:
I agree but right now passing the config itself for just picking a single consensus-independent option felt wrong too, especially because I didn't want this new option to be part of the cosmos driven config.

Reply:
That's fair, but the `CosmosSwingsetConfig` vs. `SwingSetConfig` problem still exists. I guess I'm fine with this refactoring if it doesn't lose the comment (or a variation thereof).